src/Generation/ResourcesGenerator.php

```diff
@@ -33,6 +33,16 @@
 
 class ResourcesGenerator
 {
+    // TODO(vNext): Remove this; only required for monolith compatibility.
+    private static function ensureDecimal(string $s): string
+    {
+        if (strpos($s, '.') === false) {
```

Comment:
Would it make sense to use the conditional ?: operator here?

Reply:
Done :)